entities/library.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class Library
+  include Validation
+  include Statistics
+
+  attr_accessor :authors, :readers, :books, :orders
+
+  def initialize
+    @authors = []
+    @books = []
+    @readers = []
+    @orders = []
+
+    load_datas
+  end
+
+  def load_datas
+    @authors = YAML.load_file(AUTHORS_DB) if File.exist?(AUTHORS_DB)
```

Comment:
move YAML.load_file(AUTHORS_DB) if File.exist?(AUTHORS_DB) to separate private method and pass DB to it.